packages/providers/upload-aws-s3/package.json

```diff
@@ -46,7 +46,11 @@
     "watch": "pack-up watch"
   },
   "dependencies": {
-    "aws-sdk": "2.1472.0",
+    "@aws-sdk/client-s3": "3.433.0",
+    "@aws-sdk/lib-storage": "3.433.0",
+    "@aws-sdk/s3-request-presigner": "3.433.0",
+    "@aws-sdk/types": "3.433.0",
+    "aws-sdk": "2.1479.0",
```

Comment:
It looks like aws-sdk is only used for `packages/providers/upload-aws-s3/src/__tests__/upload-aws-s3.test.ts`, should we update those to run with `@aws-sdk/client-s3` instead, and remove the dependency? Or is it still needed somehow?

Reply:
Yep, that should be done with the new tests

Reply:
So we can remove this dependency?

Reply:
Yes, done!